### PR TITLE
fix(quick-add): fix alignment of quick add

### PR DESCRIPTION
**Pull request by SMahil**
### Mandatory
 - [ ] What does this PR do? 
This PR will fix the alignment of quick add, making consistent padding around the elements and fix visibility of the infotip icon 
- [ ] What issue/task does this PR references?
https://openshift.io/openshiftio/openshiftio/plan/detail/2512
https://openshift.io/openshiftio/openshiftio/plan/detail/2511

___
- [ ] @mention(s) to expected reviewer(s) included
@rgarg1 @nimishamukherjee

### CODE REVIEW

**Comment by vikram-raj** [comment]
[test]

**Comment by rgarg1** [comment]
Build timed out. No test failures in logs.

**Comment by jarifibrahim** [comment]
[test]

File: src/app/components_ngrx/toolbar-panel/toolbar-panel.component.html [hunk 1]
```diff
@@ -8,12 +8,12 @@
     (onFilterTypeAhead)="filterQueries($event)">
   </pfng-toolbar>
 </div>
-<div class="active-filters-section">  
+<div class="f8-filters">  
   <span class="dib margin-right-10"
     *ngIf="activeFilterFromSidePane !== ''">
     <strong>{{totalCount | async}}</strong> in {{activeFilterFromSidePanel}}
   </span>
-  <div class="active-filters-div"
```
Comment (Raunak1203) [inline src/app/components_ngrx/toolbar-panel/toolbar-panel.component.html]: @SMahil Work Item datatable list: should filter work item by type is waiting for class ".active-filters-div"
Please update this:
https://github.com/fabric8-ui/fabric8-planner/blob/master/src/tests/ui/planner/toolbarHeader.ts#L26

**Comment by alien-ike** [comment]
### Ike Plugins (test-keeper)

Thank you @SMahil for this contribution!



It appears that no tests have been added or updated in this PR.

Automated tests give us confidence in shipping reliable software. Please add some as part of this change.

If you are an admin or the reviewer of this PR and you are sure that no test is needed then you can use the command `/ok-without-tests` as a comment to make the status green.


For more information please head over to official [documentation](http://arquillian.org/ike-prow-plugins/#_test_keeper_plugin). You can find there how to configure the plugin - for example exclude certain file types so if PR contains only them it won't be checked.

**Comment by rgarg1** [comment]
@jarifibrahim @Raunak1203 Pls retrigger the build. It failed due to _rxjs_ issue, which is now resolved.

**Comment by Raunak1203** [comment]
[test]

**Comment by Raunak1203** [comment]
[test]

**Comment by Raunak1203** [comment]
[test]

File: src/app/components/work-item-quick-add/work-item-quick-add.component.less [hunk 2]
```diff
@@ -0,0 +1,128 @@
+@import (reference) '../../../assets/stylesheets/_base';
```
Comment (sanbornsen) [inline src/app/components/work-item-quick-add/work-item-quick-add.component.less]: How come this file was added back to the repo?
@SMahil you need to more careful about your rebase and resolving conflicts. You always bring back old files. Please verify your conflict resolves with someone else everytime you do it.
@vikram-raj and @nimishamukherjee I see your approval there, but I believe the rebase happened after the approval so can't really blame. 

This is not the first time, so we need to be careful about rebasing and resolving conflicts. @SMahil I recommend you to take a proper git class from somewhere.

**Pull request merged**